src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -340,7 +340,8 @@ bool InputProcessor::checkVersionMatch(EnergyPlusData &state)
                     Which = static_cast<int>(index(v, MatchVersion));
                 }
                 if (Which != 0) {
-                    ShowWarningError(state, "Version: in IDF=\"" + v + "\" not the same as expected=\"" + MatchVersion + "\"");
+                    // this is reported in GetProjectData
+                    // ShowWarningError(state, "Version: in IDF=\"" + v + "\" not the same as expected=\"" + MatchVersion + "\"");
```

Comment:
I am seeing duplicate messages so either the version issue is reported here or where the version object is read. I chose to comment this out here but maybe there is a reason to do the opposite (e.g., API calls).

    ** Warning ** Version: in IDF="24.1" not the same as expected="24.2"
    ** Warning ** Version: in IDF="24.1" not the same as expected="24.2"